### DISCUSSION TITLE
How can I allow people to continue my repository?

### DISCUSSION THREAD

**Discussion by dariohsb** [community/community#86585] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I get people to continue my repository if I leave it inactive and don't want to maintain it?

**Comment by kamafozilov** [comment, marked as answer]
If you find someone who wants to continue your project, you can give them control of it. To do this, go to your project's settings on GitHub and use the option to transfer the project to their account. It's a simple process in GitHub where you just need to put in their username.

![image](https://github.com/community/community/assets/149860188/2ab6bbd1-f67b-46a2-b13a-5371f14aba63)

**Reply by dariohsb** [reply]
Thanks mate!
